### ISSUE TITLE
PII on Cloud Object Storage

### ISSUE THREAD

**Issue by rodalton** [IBM-Bluemix-Docs/cloud-object-storage#45]
The Tip around PII on this page is misleading https://console.bluemix.net/docs/services/cloud-object-storage/getting-started-cli.html#getting-started-cli-

Suggests PII cannot be stored on IBM COS when in fact it's the name of the bucket/object that should not contain PII.

**Comment by nglange** [comment]
Good point.  That language came from legal, but I'll see if we can change it to:

> Personally Identifiable Information (PII): When creating buckets and/or adding objects, please ensure to not use any information that can identify any user (natural person) by name, location or any other means in the name of the bucket or object.

**Comment by coolmary** [comment]
Thanks for the report. I'm very please for your Generosity and Loyalty response. P.S. Have a wonderful Holiday Cheers GitHub Seasons Greetings. From Mary Ann Sanford Robinson and Fitzgerald Jerome Robinson
